pkgs/by-name/sp/space-grotesk/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "space-grotesk";
+  version = "2.0.0";
+
+  src = fetchzip {
+    url = "https://github.com/floriankarsten/space-grotesk/releases/download/${version}/SpaceGrotesk-${version}.zip";
+    stripRoot = false;
+    hash = "sha256-niwd5E3rJdGmoyIFdNcK5M9A9P2rCbpsyZCl7CDv7I8=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/fonts/truetype
+    install -Dm644 SpaceGrotesk-${version}/ttf/static/*.ttf $out/share/fonts/truetype/
+
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://floriankarsten.github.io/space-grotesk";
+    description = "A proportional sans-serif typeface retaining monospace's idiosyncratic details while optimizing for improved readability";
```

```suggestion
    description = "Proportional sans-serif typeface retaining monospace's idiosyncratic details while optimizing for improved readability";
```
Not start with the definite or an indefinite article.